mods/sgssb/moves.js

```diff
@@ -602,6 +602,7 @@ exports.BattleMovedex = {
 		id: "wreakhavoc",
 		isNonstandard: true,
 		name: "Wreak Havoc",
+		desc: "Changes the pokemon's ability to Virus.",
```

Comment:
shortDesc is used in /data

Reply:
Well, ``desc`` will work from my research, as long as there is no ``shortDesc`` in which case the shortDesc is preferred for /data.  The ``desc`` is preferred for battle descriptions